### PR TITLE
Update reloader_plugin.py

### PR DESCRIPTION
**Pull request by All4Gis**
Remove `QgsMessageBar` and use 

> from qgis.core import Qgis as QGis

 for **QGIS 3**

### CODE REVIEW

File: reloader_plugin.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 from qgis.PyQt.QtGui import *
 from qgis.PyQt.QtWidgets import *
 from qgis.PyQt import uic
-from qgis.gui import QgsMessageBar
+from qgis.core import Qgis as QGis
```
Comment (borysiasty) [inline reloader_plugin.py:23]: Hi, why not just import Qgis under its normal name?
Sorry for the late reply, I was on a holiday last week.

Comment (All4Gis) [reply reloader_plugin.py:23]: it's a habit, for use the same code in a QGIS 2 and 3 and only change the imports
For example:

    try:
        from qgis.core import Qgis as QGis
        QGIS_VERSION = 'QGIS3'
    except ImportError:
        from qgis.core import QGis
        QGIS_VERSION = 'QGIS2'

Comment (borysiasty) [reply reloader_plugin.py:23]: Ahhh, forgot it's a dual-version plugin. Too much work this week ;)
Thanks a lot!

Comment (borysiasty) [reply reloader_plugin.py:23]: Actually it isn't :) Merged anyway, just releasing.

**Pull request merged**